mne_bids/tests/test_write.py

```diff
@@ -1400,16 +1400,16 @@ def test_write_raw_pathlike():
     raw = _read_raw_fif(raw_fname)
 
     bids_root = Path(_TempDir())
-    events_fname = (Path(data_path) / 'MEG' / 'sample' /
-                    'sample_audvis_trunc_raw-eve.fif')
+    events_fname = Path(data_path) / 'MEG' / 'sample' /\
+        'sample_audvis_trunc_raw-eve.fif'
```

Comment:
please avoid cosmetic changes

Reply:
My pylinter said it was causing a pep error so I was trying to change it so that it's compatible with the latest version